app/src/main/AndroidManifest.xml

```diff
@@ -14,16 +14,21 @@
   ~ limitations under the License.
   -->
 
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <!-- TODO: Remove when project updates to minSdk=24
```

Comment:
fyi @satishshendeg 

This error came up when upgrading AGP to 8.5, so I had to include this `overridesLibrary` rule since draganddrop has minSdk=24 , and this project is currently minSdk=21